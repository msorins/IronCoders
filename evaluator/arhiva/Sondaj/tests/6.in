1000000 1000100