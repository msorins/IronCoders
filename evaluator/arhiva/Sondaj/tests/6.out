1000008 96 1
